### ISSUE TITLE
Allow the ability to update the TEST_UID and FIREBASE_AUTH_JWT during a test (if needing to login with other users)

### ISSUE THREAD

**Issue by dhair-seva** [prescottprue/cypress-firebase#70]
I have been needing the ability to update the FIREBASE_AUTH_JWT (calling createTestEnvFile) during a test. I think the best way to do this is the move the createTestEnvFile into a task. This would allow us to still use node.js, but still have all the functionality there. Can this be possible?

**Comment by dhair-seva** [comment]
Here is a reference I used that gave me this idea.
https://glebbahmutov.com/blog/powerful-cy-task/

**Comment by prescottprue** [comment]
Yes I really like this idea a lot, thanks for posting! I actually was considering this as a way to support adding login for multiple users by UID (as mentioned in #14 ).

**Comment by dhair-seva** [comment]
That would be amazing to have. Now thinking about it, it might be more valuable to directly use your command, kind of like how you use cy.callFirestore. At that point, we could do cy.exec("${npm bin}/createTestEnvFile"). Then we can pass in the uid into this command. Anyway, those are some ideas. After Christmas, I will have more time to help contribute when these type of issues come up. But for now, I can only give ideas since I don't have as much time as I would like.

**Comment by dhair-seva** [comment]
@prescottprue I tried doing my last comment with this code:
```
Cypress.env("TEST_UID", CypressConfig.users[role].uid)
return cy
  .exec(`set TEST_UID=${CypressConfig.users[role].uid}`)
  .exec("npm run e2e:set:config", { env: process.env }).its("stdout").should("contain", "cypress.env.json updated successfully")
```
My script in package.json is:
```
./node_modules/.bin/cypress-firebase createTestEnvFile
```

I also tried the second exec command to be `node_modules\\.bin\\cypress-firebase createTestEnvFile`. Both commands shut down cypress' browser with no error. If I comment out the second exec command, it runs just fine. It seems that it fails only when I'm running that command. I can run, for example, `npm run lint` (I know, I would never run that in cypress, but just as a test).

Any ideas why it would fail with createTestEnvFile?

**Comment by dhair-seva** [comment]
Doesn't this issue get resolved with 0.10.0?

**Comment by prescottprue** [comment]
@dhair-seva Yes it does - thanks for posting.

I guess it doesn't necessarily update the value in the environment as described by the title, but a new JWT is generated when passing a UID to `cy.login`, it just isn't written to the environment.
